package.json

```diff
@@ -8,7 +8,13 @@
   },
   "scripts": {
     "start": "gulp compile && gulp theme",
-    "postinstall": "./scripts/pattern_lab.sh && ./scripts/twig_functions.sh"
+    "pl": "node scripts/node/pattern_lab.js",
```

Comment:
fwiw you can make node files executable by beginning the file with the heading `#!/usr/bin/env node` and calling them as `./scripts/script.js` after making them executable via `chmod +x ./scripts/script.js`

Reply:
@flipactual When I do this and run the scripts I get the error `Error: Cannot find module 'shelljs'`